### ISSUE TITLE
Use https://github.com/dave/jsgo for hosting the JS

### ISSUE THREAD

**Issue by johanbrandhorst** [johanbrandhorst/grpcweb-example#16]
Since this example runs in the _real world_, it could benefit from the CDN caching provided by https://github.com/dave/jsgo.

**Comment by johanbrandhorst** [comment]
See https://github.com/dave/ebiten/blob/master/examples/2048/index.jsgo.html for an example of a loading bar.

**Comment by johanbrandhorst** [comment]
Could potentially automate publishes: 

```bash
go get -u github.com/alexanderGugel/wsd
wsd -origin=https://compile.jsgo.io/ -url=wss://compile.jsgo.io/_ws/johanbrandhorst/grpcweb-example/client
```

**Comment by johanbrandhorst** [comment]
Fixed by  17eca2c
